natpl/src/runtime.rs

```diff
@@ -215,16 +203,10 @@ impl Runtime {
                 }
             }
             Item::VariableDeclaration { fc: _, name, rhs } => {
-                let value = self.eval_expr(&rhs, call_stack)?;
-
                 let name = name.name();
-                match self.variables.entry(name.clone()) {
-                    Entry::Occupied(_) => Err(ItemError::VariableRedefined(name)),
```

Comment:
Is this error used anywhere else now? If no then it could be removed too I think

Reply:
oh it's not, i was relying on the `dead_code` lint. i'll remove it